Ray.jl/src/runtime.jl

```diff
@@ -0,0 +1,145 @@
+    --node-ip-address=127.0.0.1 --metrics-export-port=60404 --dashboard-agent-port=60493 \
+    --listen-port=52365 --node-manager-port=58888 \
+    --object-store-name=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/sockets/plasma_store \
+    --raylet-name=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/sockets/raylet \
+    --temp-dir=/tmp/ray --session-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385 \
+    --runtime-env-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/runtime_resources \
+    --log-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/logs \
+    --logging-rotate-bytes=536870912 --logging-rotate-backup-count=5 \
+    --session-name=session_2023-08-14_14-54-36_055139_41385 \
+    --gcs-address=127.0.0.1:6379 --minimal --agent-id 470211272"
+    =#
+    line = open("/tmp/ray/session_latest/logs/raylet.out") do io
+        while !eof(io)
+            line = readline(io)
+            if contains(line, "Starting agent process")
+                return line
+            end
+        end
+    end
+
+    # --raylet-name=/tmp/ray/session_2023-08-14_18-52-23_003681_54068/sockets/raylet
+    raylet_match = match(r"raylet-name=((\/[a-z,0-9,_,-]+)+)", line)
+    raylet = raylet_match !== nothing ? String(raylet_match[1]) : error("Unable to find Raylet socket")
+
+    # --object-store-name=/tmp/ray/session_2023-08-14_18-52-23_003681_54068/sockets/plasma_store
+    store_match = match(r"object-store-name=((\/[a-z,0-9,_,-]+)+)", line)
+    store = store_match !== nothing ? String(store_match[1]) : error("Unable to find Object Store socket")
+
+    # --gcs-address=127.0.0.1:6379
+    gcs_match = match(r"gcs-address=(([0-9]{1,3}\.){3}[0-9]{1,3}:[0-9]{1,5})", line)
+    gcs_address = gcs_match !== nothing ? String(gcs_match[1]) : error("Unable to find GCS address")
+
+    # --node-ip-address=127.0.0.1
+    node_ip_match = match(r"node-ip-address=(([0-9]{1,3}\.){3}[0-9]{1,3})", line)
+    node_ip = node_ip_match !== nothing ? String(node_ip_match[1]) : error("Unable to find Node IP address")
+
+    # --node-manager-port=63639
+    port_match = match(r"node-manager-port=([0-9]{1,5})", line)
+    node_port = port_match !== nothing ? parse(Int, port_match[1]) : error("Unable to find Node Manager port")
+
+    # TODO: downgrade to debug
+    @info "Raylet socket: $raylet, Object store: $store, Node IP: $node_ip, Node port: $node_port, GCS Address: $gcs_address"
+
+    return (raylet, store, gcs_address, node_ip, node_port)
+end
+
+# TODO: use something like the java config bootstrap address (?) to get this
+# information instead of parsing logs?  I can't quite tell where it's coming
+# from (set from a `ray.address` config option):
+# https://github.com/beacon-biosignals/ray/blob/beacon-main/java/runtime/src/main/java/io/ray/runtime/config/RayConfig.java#L165-L171
+initialize_coreworker() = initialize_coreworker(bootstrap_ray_args_from_raylet_out()...)
+initialize_coreworker(args...) = rayjll.initialize_coreworker(args...)
```

Comment:
```suggestion
initialize_coreworker() = rayjll.initialize_coreworker(bootstrap_ray_args_from_raylet_out()...)
```

unless we need the method with `args`?

Reply:
yeah good question; I thnk we probably _don't_ need it for now.